### PR TITLE
cdc: canal-json dml event commit-ts should be tso

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-canal-json.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ TiCDC 会把一个 DDL Event 编码成如下 Canal-JSON 格式：
     ],
     "old": null,
     "_tidb": {     // TiDB 的扩展字段
-        "commitTs": 163963314122145239
+        "commitTs": 429918007904436226  // tidb tso
```
Comment (Oreoxmt) [inline ticdc/ticdc-canal-json.md:132]: @3AceShowHand Could you please confirm if the following documents also need to be updated:

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-canal-json.md?plain=1#L62-L63

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-sink-to-kafka.md?plain=1#L456-L457

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-sink-to-kafka.md?plain=1#L522-L523

Comment (Oreoxmt) [inline ticdc/ticdc-canal-json.md:132]:
```suggestion
        "commitTs": 429918007904436226  // TiDB TSO 时间戳
```

File: ticdc/ticdc-canal-json.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ WATERMARK Event 的示例如下：
     "data": null,
     "old": null,
     "_tidb": {     // TiDB 的扩展字段
-        "watermarkTs": 429918007904436226
+        "watermarkTs": 429918007904436226  // tidb tso
```
Comment (Oreoxmt) [inline ticdc/ticdc-canal-json.md:161]:
```suggestion
        "watermarkTs": 429918007904436226  // TiDB TSO 时间戳
```

File: ticdc/ticdc-canal-json.md (hunk 1 above)
Comment (3AceShowHand) [reply ticdc/ticdc-canal-json.md:132]: Yes, they also need to be updated.

**Review by Oreoxmt — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -454,7 +454,7 @@ large-message-handle-option = "handle-key-only"
     ],
     "old": null,
     "_tidb": {     // TiDB 的扩展字段
-        "commitTs": 163963314122145239,
+        "commitTs": 429918007904436226  // TiDB TSO 时间戳
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:457]:
```suggestion
        "commitTs": 429918007904436226,  // TiDB TSO 时间戳
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -520,7 +520,7 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:523]:
```suggestion
        "commitTs": 429918007904436226,  // TiDB TSO 时间戳
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -454,7 +454,7 @@ large-message-handle-option = "handle-key-only"
     ],
     "old": null,
     "_tidb": {     // TiDB 的扩展字段
-        "commitTs": 163963314122145239,
+        "commitTs": 429918007904436226,  // TiDB TSO 时间戳
```
Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:457]:
```suggestion
        "commitTs": 429918007904436226  // TiDB TSO 时间戳
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -520,7 +520,7 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
```
Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:523]:
```suggestion
        "commitTs": 429918007904436226  // TiDB TSO 时间戳
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**